README.md

```diff
@@ -1,36 +1,5 @@
 <a name="readme-top"></a>
 
```

Comment:
- [ ] In the configureStore file you need to import `configureStore` from `@reduxjs/toolkit` and use it to set up your store. Kindly refactor you code accordingly.

- [ ] You are required to also import the categories reducer in the file and add it as a reducer in the store.

  ![image](https://user-images.githubusercontent.com/54848363/215206533-3a97e2bc-36f7-4df7-9288-766da96c658f.png)